wiki/Client/Beatmap_editor/Menu/de.md

```diff
@@ -1,178 +1,92 @@
----
-
 # Menü des Beatmap-Editors
 
 ## Datei
 
 ![Dateimenü](img/M_File-DE.jpg "Dateimenü")
 
-**Grundlegende Datei-Befehle.**
-
-### Datei-Befehle
-
 | Name | Beschreibung |
 | :-- | :-- |
 | Alle Noten löschen (`Strg` + `N`) | Entfernt alle Hit-Objekte der Schwierigkeitsstufe. |
-| Schwierigkeitsstufe öffnen... | **Zum Editieren:** Wechsle schnell zwischen den Schwierigkeitsstufen. **Zur Referenz:** Lege die gewählte Schwierigkeitsstufe über die (aktuell) geöffnete (nur osu!mania). |
-| Speichern (`Strg` + `S`) | Speichert alle Änderungen (Dateiformate `.osu` und `.osb`). |
-| Neue Schwierigkeitsstufe erstellen... | Speichere die aktuellen Änderungen dieser Schwierigkeitsstufe als eine neue Stufe. Die alte Schwierigkeitsstufe bleibt beim erneuten Laden in ihrem zuletzt gespeicherten Zustand erhalten. |
-
-### Befehle zum Zurücksetzen
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Zum Gespeicherten zurücksetzen (`Strg` + `L`) | Hebt alle Änderungen auf und kehrt zum zuletzt gespeicherten Zustand zurück. Für kleine Fehler (Hit-Objekte). |
-| Zum Gespeicherten zurücksetzen (Komplett) (`Strg` + `Shift` + `L`) | Hebt alle Änderungen auf und kehrt zum zuletzt gespeicherten Zustand zurück. Für komplizierte Fehler (Timing, Storyboard). |
-
-### Befehle zur Fehlerbehebung
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Beatmap testen (`F5`) | Teste die Beatmap. Aktuelle Änderungen werden für den Test gespeichert. |
-| AiMod öffnen (`Strg` + `Shift` + `A`) | AiMod öffnen. |
-
-### Befehle zum Verpacken von Beatmaps
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Beatmap hochladen ... (`Strg` + `Shift` + `U`) | Lädt die Beatmap in die Sektion [WIP](https://osu.ppy.sh/community/forums/10) hoch. Du musst mit deinem Account auf der Webseite angemeldet sein, um das Beatmap-Forum sehen zu können. |
-| Paket exportieren ... | Exportiert die Beatmap als `.osz`. Die Datei wird zum Ordner "Exports" gesendet und der Ordner wird direkt danach geöffnet. |
-| Map-Paket entpacken | ? |
-| Importieren von ... | **bms/bme:** (nur für osu!mania) Öffnet .bms/.bme Dateien, um sie als osu!mania-Schwierigkeitsstufe zu importieren. |
-
-### Befehle für Programmierer
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Songs-Ordner öffnen | Öffnet den Song-Ordner, welcher sich im Verzeichnis osu!/Songs befindet. |
-| `.osu` im Editor öffnen | Öffnet die Schwierigkeitsstufe im Texteditor. |
-| `.osb` im Editor öffnen | Öffnet die Storyboard-Datei im Texteditor. |
-
-### Befehl für den Notausstieg
-
-| Name | Beschreibung |
-| :-- | :-- |
-| Beenden... (`Esc`) | Beatmap-Editor verlassen. Du wirst möglicherweise gefragt, ob du kürzliche Änderungen speichern möchtest. *Ignoriert* manchmal Änderungen, die im Design-Tab gemacht wurden. |
+| Schwierigkeitsstufe öffnen ... | **Zum Editieren:** Wechsle schnell zwischen den Schwierigkeitsstufen. **Zur Referenz:** Lege die gewählte Schwierigkeitsstufe über die (aktuell) geöffnete (nur [osu!mania](/wiki/Game_mode/osu!mania)). |
```

Comment:
Some of these use an imperative form ("Wechsle", "Speichere") while some use the third person ("[Diese Schaltfläche[^1]] Speichert alle Änderungen"). Is this on purpose? The english version seems to always use the imperative form ("Remove all hit objects" rather than "Removes all hit objects").

[^1]: or "Ein Klick auf den Knopf" etc. You know what I mean

Reply:
Some may sound better in imperative, some are succinct in third person 🤷 

Example:

- Imperative: I would prefer `Teste die Beatmap` over `Testet die Beatmap` --> Requires a change to e.g. `Startet den Testmodus`
- 3rd person: I prefer `Ausgewählte Objekte ausschneiden.` over `Schneide die ausgewählten Objekte aus.`

If we decide to change them to only imperative or vice versa, I would need to change the form regardless. Just switching the verb does not always work like in the first example.

There are only a handful of sentences that still use imperative. Do you want to me change those to third person?

So if there's one direction to go, then I would suggest the 3rd person.

Reply:
Hm.. on second thought, it's more important to have something that reads well than to have perfect consistency, at least IMHO. Let's maybe try and make it more consistent (3rd person sounds fine) while not over-correcting stuff that works well. Sound alright?